src/utils/deprecate.js

```diff
@@ -25,7 +25,7 @@ const deprecate = function(message, test) {
 
 deprecate._console = typeof console !== 'undefined' ? console : {};
 deprecate._warn = function() {
-  const warn = deprecate._console.warn || deprecate._console.log || function() {};
+  const warn = deprecate._console.warn || deprecate._console.log || Function.prototype;
```

Comment:
@JSteunou no) I just like `Function.prototype` more then `function() {}`. I can return how it was.